src/main/java/com/g4s8/ghman/bot/Notification.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019 Kirill Ch. (g4s8.publci@gmail.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to read
+ * the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.g4s8.ghman.bot;
+
+import com.g4s8.ghman.user.Thread;
+import com.g4s8.ghman.user.Users;
+import com.g4s8.teletakes.bot.BotSimple;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import org.telegram.telegrambots.api.methods.send.SendMessage;
+import org.telegram.telegrambots.exceptions.TelegramApiException;
+
+/**
+ * Notify user into telegram chat.
+ * @since 1.0
+ */
+public interface Notification {
+
+    /**
+     * Sends notification.
+     * @throws IOException When fails
+     */
+    void send() throws IOException;
+
+    /**
+     * Unread threads notification.
+     * @since 1.0
+     * @todo #9:30 Continue implement notifications for user when new messages
+     *  (unread github notification) arrive. Unread msgs can be retrieved with
+     *  the help of PgThreads#unread() method, and this class is meant to sent
+     *  them to tlg chat. So set up whole mechanism together.
+     * @todo #9:30min Now this class send only github subject title as
+     *  notification. Please aks ARC what exactly we want to send to user and
+     *  create a class to extract and format this information from `JsonObject`
+     *  (github subject) accordingly.
+     */
+    class UnreadThreads implements Notification {
+
+        /**
+         * Telegram bot.
+         */
+        private final BotSimple bot;
+
+        /**
+         * Users and notification map.
+         */
+        private final Map<Long, List<Thread>> ntfc;
+
+        /**
+         * Users.
+         */
+        private final Users users;
+
+        /**
+         * Ctor.
+         * @param bot Telegram bot
+         * @param ntfc Notifications list
+         * @param users Users
+         */
+        public UnreadThreads(final BotSimple bot,
+            final Map<Long, List<Thread>> ntfc, final Users users) {
```

Comment:
@olenagerasimova why doesn't this notification directly takes `PgThreads` and call `unreads()` itself when `send()` is called? It would be more OO (even though I'm not sure what you had in mind).

Reply:
@victornoel I do not like the idea of passing `PgThreads` to this class, because `PgThreads` does not implement any interface. Now it's a class that depends on database source and we definitely do not want deal with it in unit tests.

Reply:
@olenagerasimova good point. Maybe @g4s8 you have an opinion if a todo should be added for introducing an interface for `PgThreads` or not?

Reply:
@victornoel I think we can merge this part now, but you can submit a ticket with refactoring suggestion if you see how to make it better

Reply:
@g4s8 yes, it's maybe too early to know what is the best course of action anyway :)